env.example

```diff
@@ -306,10 +306,13 @@ JIBRI_FINALIZE_RECORDING_SCRIPT_PATH=/config/finalize.sh
 # XMPP user for Jibri client connections
 JIBRI_XMPP_USER=jibri
 
-# MUC name for the Jibri pool
+# MUC name for the Jibri pool. The JID of the MUC to be used as a brewery for jibri instances for streaming.
```

Comment:
nit: It's just the name of the room ("JibriBrewery") as opposed to the JID ("JibriBrewery@something").

Reply:
sure I will update it